core/chain_indexer.go

```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/dominant-strategies/go-quai/params"
 )
 
-var PruneDepth = uint64(1000) // Number of blocks behind in which we begin pruning old block data
+var PruneDepth = uint64(1000000) // Number of blocks behind in which we begin pruning old block data
```

Comment:
At 10k blocks a day this means we don't prune for 100 days. The re-org tolerance depth is well before this timeframe